common.js

```diff
@@ -1943,6 +1943,251 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Subaru': {
+    'BRZ': {
+      '2013': ['stx', 'csp', 'dsp', 'ssc'],
+      '2014': ['stx', 'csp', 'dsp', 'ssc'],
+      '2015': ['stx', 'dsp', 'ssc'],
+      '2016': ['stx', 'dsp', 'ssc'],
+      '2017': ['stx', 'dsp'],
+      '2018': ['stx', 'dsp'],
+      '2019': ['stx'],
+      '2020': ['stx'],
+    },
+    'BRZ Series.Blue': {
+      '2015': ['ssc'],
+    },
+    'BRZ Series.HyperBlue': {
+      '2016': ['ssc'],
+    },
+    'BRZ tS': {
+      '2018': ['cs'],
+    },
+    'BRZ (non-Performance Package)': {
```

Comment:
Instead of having `BRZ`, `BRZ non-performance` and `BRZ performance` models be separate, can you just have BRZ non performance and BRZ performance and add the rest of the classes from the regular BRZ to both?

Also, the BRZ non-performance needs to include `ssc` class. Thanks!